app/main.py

```diff
@@ -1,3 +1,44 @@
+from __future__ import annotations
+
+
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: int | float) -> None:
+        self.km = km
+
+    @staticmethod
+    def get_km(distance: Distance | int | float) -> int | float:
+        return distance.km if isinstance(distance, Distance) else distance
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: Distance | int | float) -> Distance:
+        return Distance(self.km + Distance.get_km(other))
+
+    def __iadd__(self, other: Distance | int | float) -> Distance:
+        self.km += Distance.get_km(other)
+        return self
+
+    def __mul__(self, other: int | float) -> Distance:
+        return Distance(self.km * other)
+
+    def __truediv__(self, other: int | float) -> Distance:
+        return Distance(round(self.km / other, 2))
+
+    def __lt__(self, other: Distance | int | float) -> bool:
+        return Distance.get_km(self) < Distance.get_km(other)
```

Comment:
you must use `self.km` instead of `Distance.get_km(self)` and to use `get_km()` form instance: `self..get_km(other)` instead of `Distance.get_km(other)`